Articles/WhatIs/WhatIsPerformanceAtLCF.md

```diff
@@ -0,0 +1,19 @@
+### What are Performance at Leadership Computing Facilities?
```

Comment:
```suggestion
### What is Performance in Scientific Computing?
```

Do we really want to talk about what our LCF's think performance is?  Or, what our community think's performance is.